foreshadow/utils/common.py

```diff
@@ -199,3 +200,30 @@ def should_force_reresolve_based_on_override(self, X):
                         return True
 
         return False
+
+
+class DataSeriesSelector(BaseEstimator, TransformerMixin):  # noqa
+    """For data in a data frame column, extract the data series.
+
+    The data is expected to be stored in a data frame column and can
+    be extracted through the column name as the key.
+
+    Parameters
+    ----------
+    column_name : the name of the column
+    """
+
+    def __init__(self, column_name):
+        self.column_name = column_name
+
+    def fit(self, X, y=None):  # noqa
+        return self
+
+    def transform(self, X):  # noqa
+        if isinstance(X, DataFrame):
+            X["concat_col"] = X.apply(
+                lambda row: " ".join(row.dropna().tolist()), axis=1
```

Comment:
What is this trying to do?

Reply:
Merge all the text columns with a space.